pnc-mock/src/main/java/org/jboss/pnc/mock/model/builders/ArtifactBuilder.java

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Yes, we probably gonna end up with *-mock modules to avoid circular dependencies.